configs/monado-vulkan.yaml

```diff
@@ -1,12 +1,7 @@
 plugin_groups:
+  - !include "rt_slam_plugins.yaml"
```

Comment:
did you miss audio?
In master it is added as a part of core_plugins.yaml

Reply:
I added them back in. Had to remove timewarp_gl from core plugins.